tools/stonith_admin.c

```diff
@@ -497,33 +497,23 @@ validate(stonith_t *st, const char *agent, const char *id,
     return rc;
 }
 
+static pcmk__supported_format_t formats[] = {
+    { "text", pcmk__mk_text_output, pcmk__text_output_entries },
+    { "xml", pcmk__mk_xml_output, pcmk__xml_output_entries },
```

Comment:
Maybe even add convenience macros for these like PCMK__SUPPORTED_FORMAT_TEXT

Reply:
Yeah, maybe.  I have similar code on my branch that does a lot of work converting crm_mon to use formatted output.  I intended to link to that here to mention hoiw I'd need to reconcile the two approaches, however I spent all of yesterday afternoon rebasing those patches work continue to work against master and it's still not done.

Reply:
Here we go - https://github.com/clumens/pacemaker/commit/a2899d74aefd51a1356ab15eb2ffe5be676dabd8 does some similar stuff, so that code and this will have to be reconciled.